packages/api/src/schema/db-schema.yaml

```diff
@@ -744,6 +744,22 @@ components:
         objectStoreId:
           type: string
           example: D8321C3E-B29C-45EB-A1BB-A623D8BE0F65
+        multistream:
+          type: object
+          additionalProperties: false
+          properties:
+            targets:
+              type: array
+              index: true
+              indexType: gin
```

Comment:
FTR as we chatted offline: these are the only fields that we need to have in this `db-schema` file, while all the rest can be kept in the `api-schema` (i.e. no changes there). These files are deep merged so the fields added here simply override the fields in the `api-schema`.

That also means that there's no problem if we repeat the whole field definition to add the index fields here, but I generally try to avoid the redundancy and add here only the fields that can't go on the OpenAPI schema (our custom stuff)

There's a `README` in this directory as well with some more info.